### PR TITLE
Update channel subscription management endpoints

### PR DESCRIPTION
**Pull request by ftcvlad**
### Description of Changes

Manage channels based on user identifier instead of install identifier. Use new endpoints that expect user identifier.

### Breaking Changes

- None

### Release Checklist

Prepare:

- [x] Detail any breaking changes. Breaking changes require a new major version number
- [x] Check `./gradlew assemble` passes w/ no errors

Bump versions in:

- [x] README.md

Release:

- [ ] Squash and merge to master
- [ ] Delete branch once merged
- [ ] Create tag from master matching chosen version

### CODE REVIEW

File: kumulos/src/main/java/com/kumulos/android/Kumulos.java [hunk 1]
```diff
@@ -60,6 +60,7 @@
     /** package */ static final String PUSH_BASE_URL = "https://push.kumulos.com";
     /** package */ static final String EVENTS_BASE_URL = "https://events.kumulos.com";
     /** package */ static final String CRM_BASE_URL = "https://crm.kumulos.com";
+    /** package */ static final String CORE_BASE_URL = "https://stats.kumulos.com";
```
Comment (cgwyllie) [inline kumulos/src/main/java/com/kumulos/android/Kumulos.java:63]: Can we use the CRM base URL for these requests?

File: kumulos/src/main/java/com/kumulos/android/PushSubscriptionManager.java [hunk 2]
```diff
@@ -71,9 +71,9 @@ public void subscribe(Context c, String[] uuids, final Kumulos.Callback callback
         }
 
         OkHttpClient httpClient = Kumulos.getHttpClient();
-        String installId = Kumulos.getInstallId();
+        String userIdentifier = Kumulos.getCurrentUserIdentifier(c);
 
-        String url = Kumulos.PUSH_BASE_URL + "/v1/app-installs/" + installId + "/channels/subscriptions";
+        String url = Kumulos.CORE_BASE_URL + "/v1/app-users/" + userIdentifier + "/channels/subscriptions";
```
Comment (cgwyllie) [inline kumulos/src/main/java/com/kumulos/android/PushSubscriptionManager.java:76]: We should probably consider URL encoding of the identifier (re previous issues on in-app sync etc.)

Comment (cgwyllie) [reply kumulos/src/main/java/com/kumulos/android/PushSubscriptionManager.java:76]: Seeing a few uses, perhaps we should abstract some URL builder func / helper?

File: kumulos/src/main/java/com/kumulos/android/Kumulos.java (hunk 1 above)
Comment (ftcvlad) [reply kumulos/src/main/java/com/kumulos/android/Kumulos.java]: k

File: kumulos/src/main/java/com/kumulos/android/PushSubscriptionManager.java (hunk 2 above)
Comment (ftcvlad) [reply kumulos/src/main/java/com/kumulos/android/PushSubscriptionManager.java]: k

**Pull request merged**